tests/toolkit/timeseries_tests.cc

```diff
@@ -0,0 +1,27 @@
+#include <gtest/gtest.h>
+
+#include "agent.h"
+
+#include "../test_context.h"
+#include "../test_agents/test_agent.h"
+
+namespace cyclus {
+namespace toolkit {
+
+TEST(TimeSeriesTests, Power) {
+  TestContext tc;
+  Agent* a = new TestAgent(tc.get());
+  RecordTimeSeries<POWER>(a, 42.0);
+  delete a;
+}
+
+TEST(TimeSeriesTests, RawPower) {
+  TestContext tc;
+  Agent* a = new TestAgent(tc.get());
+  RecordTimeSeries<double>("Power", a, 42.0);
+  delete a;
+}
```

Comment:
In my opinion, the existence of a valid test is always better than no test, irrespective of the depth of functionality it tests. Unit tests in particular, should aim to test as simple of a behavior as possible.  I think that it is also worth noting that these underlying functions don't actually do that much and aren't that big.  They exist to establish a convention, not implement a lot of behavior.  I think that this is actually a pretty decent example of a small function with small tests implemented via mocked behavior.